### PR TITLE
Lore Overhaul

### PR DESCRIPTION
**Pull request by Novacat**

### CODE REVIEW

**Comment by KasparoVy** [comment]
Hackerman hacking us back in time, hell yeah.

File: code/modules/games/spaceball_cards.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	..()
 	var/datum/playingcard/P
 	var/i
-	var/year = 554 + text2num(time2text(world.timeofday, "YYYY"))
+	var/year = 310 + text2num(time2text(world.timeofday, "YYYY"))
```
Comment (Arokha) [inline code/modules/games/spaceball_cards.dm:11]: I get the feeling this was originally an oversight and you probably shouldn't keep it 10 years in the future of the present. They probably don't make playing cards for sports teams 10 years in the future of the current date!

File: code/game/objects/banners_vr.dm [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/obj/item/weapon/banner/solgov
+	name = "\improper SolCom banner"
```
Comment (Arokha) [inline code/game/objects/banners_vr.dm:2]: Doesn't this need, like, an icon state and stuff

File: vorestation.dme [hunk 3]
```diff
@@ -1028,6 +1029,8 @@
 #include "code\game\objects\effects\decals\Cleanable\tracks.dm"
 #include "code\game\objects\effects\decals\posters\bs12.dm"
 #include "code\game\objects\effects\decals\posters\polarisposters.dm"
+#include "code\game\objects\effects\decals\posters\polarisposters_vr.dm"
+#include "code\game\objects\effects\decals\posters\tgposters.dm"
```
Comment (Arokha) [inline vorestation.dme:1033]: Huh, tgposters wasn't checked?

Comment (Novacat) [reply vorestation.dme:1033]: Nope! I decided to check it when I saw that oversight.

File: code/game/objects/banners_vr.dm (hunk 2 above)
Comment (Novacat) [reply code/game/objects/banners_vr.dm:2]: It is in banner.dm

**Pull request merged**